contracts/build.rs

```diff
@@ -17,7 +17,11 @@ fn main() {
 
     generate_contract("IERC20", hashmap! {});
     generate_contract("ERC20Mintable", hashmap! {});
-    generate_contract("UniswapV2Router02", hashmap! {});
+    // TODO: make a real deployment of the contract and set the address here
+    generate_contract(
+        "UniswapV2Router02",
+        hashmap! {4 => Address::from_str("0000000000000000000000000000000000000000").unwrap()},
```

Comment:
Let's use the already deployed version at https://rinkeby.etherscan.io/address/0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D (https://uniswap.org/docs/v2/smart-contracts/router02/)